Makefile

```diff
@@ -68,6 +68,9 @@ COLOR_FAIL=$(shell printf "\033[31mFAIL\033[0m")
 COLORIZE=$(SED) ''/PASS/s//$(COLOR_PASS)/'' | $(SED) ''/FAIL/s//$(COLOR_FAIL)/''
 DOCKER_NAMESPACE?=jaegertracing
 DOCKER_TAG?=latest
+PLATFORMS?=linux/amd64,linux/arm64
```

Comment:
We are already defining `PLATFORMS` in `docker/Makefile` and we are including that Makefile here in line no. 5, so it should be available already.

Reply:
`docker/Makefile` need self scope

`PLATFORMS` could overwrite for all.